test-case/multiple-pipeline-capture.sh

```diff
@@ -97,10 +97,12 @@ func_run_pipeline_with_type()
 func_error_exit()
 {
     dloge "$*"
-    pgrep -a aplay
-    pgrep -a arecord
-    pkill -9 aplay
-    pkill -9 arecord
+    ( set +e
+      pgrep -a aplay
+      pgrep -a arecord
+      pkill -9 aplay
+      pkill -9 arecord
+    )
```

Comment:
how about `! pgrep -a aplay ||  pkill -9 aplay`, I think this should be compatible with set -e. the (set +e ...) looks like some hack.

Reply:
I think this a good idea and better code however I don't think it's worth delaying this very urgent fix and I don't think (set +e ...) is a hack at all

Is it enough if I promise to submit a follow up commit with your change ASAP after this?

Reply:
@marc-hb sure, this is already good enough

Reply:
> how about ! pgrep -a aplay || pkill -9 aplay

#523